scripts/notify-docs/published-headless-atomic.js

```diff
@@ -0,0 +1,38 @@
+const {Octokit} = require('@octokit/rest');
+const headlessPackageJson = require('../../packages/headless/package.json')
+const atomicPackageJson = require('../../packages/atomic/package.json')
+
+const token = process.env.GITHUB_TOKEN || '';
+const github = new Octokit({ auth: token });
+
+const owner = 'coveo';
+const repo = 'doc_jekyll-public-site';
+
+async function notify(event_type, client_payload) {
+  return github.repos.createDispatchEvent({owner, repo, event_type, client_payload});
+}
+
+function logError(e) {
+  const {status, message, request} = e;
+    console.log(status, message, request);
+}
+
+async function main() {
```

Comment:
This works and is simple enough, so I'm fine with it.

I think however the "cleanest" would have been to accept which package to notify as script arguments.

```
node notify-doc.js -package headless@1.2345 -package atomic@9.876 - package quantic@3.1416 -package ...
```